geode-core/src/distributedTest/java/org/apache/geode/management/JMXMBeanReconnectDUnitTest.java

```diff
@@ -133,12 +137,14 @@ public void setUp() throws Exception {
     locator2Port = port[1];
     locator1JmxPort = port[2];
     locator2JmxPort = port[3];
-    locators = "localhost[" + locator1Port + "];localhost[" + locator2Port;
+    locators = "localhost[" + locator1Port + "],localhost[" + locator2Port + "]";
```

Comment:
This line changed. I don't think this fixed GEODE-7739, but I'm going to retest. Even if it avoids GEODE-7739 in this test, that bug still needs to be fixed either way.